zerodep/src/main/java/com/lightbend/microprofile/reactive/streams/zerodep/StageInlet.java

```diff
@@ -90,23 +90,30 @@ public void onUpstreamFailure(Throwable error) {
 
   /**
    * Indicates that an element has been pushed. The element can be received using {@link StageInlet#grab()}.
+   * <p>
+   * If this throws an exception, the error will be passed to {@link #onUpstreamFailure(Throwable)}, upstream will
```

Comment:
imprecise wording; failure passed into the next stages onUpstreamFailure, we are their upstream, not "mine" as the wording could be interpreted as.